src/components/App/App.js

```diff
@@ -1,42 +1,50 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { ThemeProvider } from 'styled-components';
 import { Route, withRouter } from 'react-router-dom';
 
 import { FRONT_END_ROLE, BACK_END_ROLE, DEV_OPS_ROLE } from '../../constants/roles';
+import { ROOT_PATH } from '../../constants/routes';
 import { location } from '../../types';
 import { GenderProvider } from '../../context/GenderContext';
 import generateTheme from '../../utils/generateTheme';
+import { RouteTransitionAnimationProvider } from '../../context/RouteTransitionAnimationContext';
```

Comment:
It is cosmetic, but i would move it one line above, to keep context providers next to each other.